src/Entity/Execution.php

```diff
@@ -0,0 +1,353 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\ExecutionRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: ExecutionRepository::class)]
+class Execution
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
+
+    #[ORM\Column(type: Types::BIGINT)]
+    private ?string $ref = null;
+
+    #[ORM\Column(length: 50, nullable: true)]
+    private ?string $filename = null;
+
+    #[ORM\Column(type: Types::DATETIME_MUTABLE, nullable: true)]
+    private ?\DateTimeInterface $start_date = null;
+
+    #[ORM\Column(type: Types::DATETIME_MUTABLE, nullable: true)]
+    private ?\DateTimeInterface $end_date = null;
+
+    #[ORM\Column]
+    private ?int $duration = null;
+
+    #[ORM\Column(length: 20)]
+    private ?string $version = null;
+
+    #[ORM\Column(length: 50)]
+    private string $campaign = 'functional';
+
+    #[ORM\Column(length: 50, nullable: false)]
+    private string $platform = 'chromium';
+
+    #[ORM\Column(nullable: true)]
+    private ?int $suites = null;
+
+    #[ORM\Column(nullable: true)]
+    private ?int $tests = null;
+
+    #[ORM\Column(nullable: true)]
+    private ?int $skipped = null;
+
+    #[ORM\Column(nullable: true)]
+    private ?int $pending = null;
+
+    #[ORM\Column(nullable: true)]
+    private ?int $passes = null;
+
+    #[ORM\Column(nullable: true)]
+    private ?int $failures = null;
+
+    #[ORM\Column(nullable: true)]
+    private ?int $broken_since_last = null;
+
+    #[ORM\Column(nullable: true)]
+    private ?int $fixed_since_last = null;
+
+    #[ORM\Column(nullable: true)]
+    private ?int $equal_since_last = null;
+
+    #[ORM\Column(type: Types::DATETIME_MUTABLE)]
+    private \DateTimeInterface $insertion_start_date;
+
+    #[ORM\Column(type: Types::DATETIME_MUTABLE)]
+    private \DateTimeInterface $insertion_end_date;
+
+    #[ORM\OneToMany(mappedBy: 'execution', targetEntity: Suite::class)]
```

Comment:
Shouldn't you add some `mappedBy` or stuff like that so that the binding between the two entities is complete?

Reply:
Fixed